source/renderer/app/components/staking/epochs/StakingEpochs.scss

```diff
@@ -1,5 +1,174 @@
+%progressLabelText {
+  font-family: var(--font-semibold);
+  font-size: 14px;
+  line-height: 24px;
+  position: absolute;
+  top: 0;
+  white-space: nowrap;
+}
+
 .component {
-  font-family: var(--font-regular);
-  margin: 20px;
-  text-align: center;
+  padding: 20px 20px 0;
+
+  .headerWrapper {
+    align-items: center;
+    display: flex;
+    justify-content: center;
+    margin: 0 20px 6px;
+  }
+
+  .epochSelector {
+    width: 275px;
+
+    input {
+      border: 0;
+      font-family: var(--font-semibold);
+      font-size: 19px;
+      line-height: 1.37;
+      padding-left: 0;
+      padding-right: 0;
+      padding-top: 0;
+    }
+  }
+
+  .noResultsLabel {
+    color: var(--theme-staking-font-color-lighter);
+    flex-grow: 1;
+    font-family: var(--font-regular);
+    font-size: 16px;
+    text-align: center;
+  }
+
+  .loadingSpinnerWrapper {
+    margin: auto;
+  }
+
+  .currentEpochProgressBar {
+    height: 24px;
+    margin-bottom: 31px;
```

Comment:
@yakovkaravelov please use `margin: ...` generic CSS rule instead of 4 specific ones 🙏